### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by loikki**
Hi,
I am thinking to use chummer in order to simplify the character generation with my friends and I thinking about doing some modifications.
From what I have seen, Arkeas is the creator of the project and he is not working on it any more, am I right? Are you still working on it or should I create my own fork and work on it?
I love to have a random system (for the main attribute) and therefore I am thinking about creating one, are you interested?
I am using the 20th anniversary edition, from what I have heard, there is a few changes with the normal one (for example in the skill groups), do you know on which edition the code is based on? EDIT: I found in the program that it should be the 4th edition (not the anniversary), therefore I will maybe do a modification in order to give the possibility to choose between the two.
An other idea (it would be a lot of work) is to convert this code to java for using it on android.

I am still waiting to know if my players are interested in it but let me know if you are interested by my possible future work :)

### CODE REVIEW

**Comment by Fizztastic** [comment]
You're indeed right, Arkeas hasn't been working on Chummer since mid-2013. I originally forked Arkeas' code to fix some issues one of my friend was having with an Infected character he was playing but I don't really plan on working on it actively unless something else comes up in our campaign. That being said, you're free to fork if you want but if you submit a Pull Request I'll review it and merge it. 

I think your idea of a random system for attribute is a great idea for NPC generation. As far as I know, the XML data is from the 20th Anniversary Edition and I'd assume Chummer uses that edition rules variations.

If you want to port the code over to Java you're more than welcome to but I personally have no plan or the knowledge to do so.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+"Chummer is a character generator for Shadowrun 4th Edition.
+Not only can you create your character quickly and easily,
+but you can also use Chummer during your character's shadowrunning career,
+to accurately track your Karma, Nuyen, ammo, and everything else all in one place.
+Chummer also includes support for a number of optional rules and house rules and even includes
+support for critters and is useful for players and Game Masters alike!
+It also supports four languages: English, French, German, and Japanese.
+
+Chummer is continuously updated to address bugs and introduce new and interesting features suggested by the community.
+If you have a bug to report or idea to suggest, please report them on the Dumpshock forums by using the link in the menu."
```
Comment (Fizztastic) [inline README.md]: Can you change the last paragraph to the Issues section of the project instead of the Dumpshock forums?

Comment (loikki) [inline README.md]: Done

**Comment by loikki** [comment]
I think it is better if we keep the number of forks still in development as low as possible in order to avoid too much chaos, therefore I will send you my modifications.

The way I was thinking the random was randomly creating the initial attributes and letting the players improving them if wanted. I have already tried a system where a player throw a dice for each attributes and if he does 5+ it increases the initial value (5->+1, 6->+2) but now I have an immortal cyber-troll therefore I still need to improve a little bit how I am doing it. [At the end of the message, I have put a text describing a little bit more].

It should be quite easy to do a random NPC character generator (at least for the attributes [and maybe skills]) and can be very useful. I will think about it.

For the android part, I will speak with my players tonight and if they are interested, I am thinking about contacting chummer5a in order to see if he can be interested by making a android app together that can create a character from the versions 4 & 5.

EDIT: I have just seen that the chummer5's guys have the 4 in their repository, maybe we can try to merge everything together

Here is the way I was doing the random:
Each player starts with 400 BP from which I am removing the theoretical average of the dice times the cost to improve it of 1 (for the system 5->+1, 6->+2, the theoretical average is 0.5 and the cost is 10, therefore I am removing 0.5_10_9[number of attributes including edge] => 400-90  = 310BP still available). For the mages/adepts/technomancer, they roll their magic attribute too, therefore the cost of the quality is increased by 0.5*10. I am keeping the limit of BP to use in attributes (I think it is 50% of the initial BP), therefore they still have 110BP to use (for normal characters [mages have a 100BP])

**Comment by loikki** [comment]
Hi,
I have spoken with the guys from chummer and I think I will still develop in your rep.
For the android part, I have maybe found something to do it easily (I will let you know if I am able to work on something)

**Pull request merged**